poem/todo-postgres/README.md

```diff
@@ -0,0 +1,8 @@
+# Postgres with Poem
+The following example is to present a way to use [poem](https://docs.rs/poem/latest/poem/) framework and devlop a relational database for a TODO app using [Postgres](https://www.postgresql.org/docs/).
+This app uses the `shuttle service` attribute and `sqlx` to build a database and define its CRUD operations.
```

Comment:
This is good, but not completely accurate. We're showing how to develop an application that *uses* a postgres database, we're not developing a database. 
```suggestion
This example shows how to develop a [poem](https://docs.rs/poem/latest/poem/) application with a [Postgres](https://www.postgresql.org/docs/) database, provisioned with shuttle.
```